### PR TITLE
Adding methods to check numerical bounds for JuMP model

### PR DESCRIPTION
**Pull request by sourabhdalvi**

### CODE REVIEW

File: src/jump_model_utils.jl [hunk 1]
```diff
@@ -0,0 +1,147 @@
+########### JuMP model utils #########
+
+mutable struct NumericalBounds
+    min::Float64
+    max::Float64
+    min_index::Any
+    max_index::Any
+end
+
+NumericalBounds() = NumericalBounds(Inf, -Inf, nothing, nothing)
```
Comment (jd-lara) [inline src/jump_model_utils.jl:10]: We should add at the top of the file that is based on the code in SDDP and reference the file and commit.

**Review by jd-lara — CHANGES_REQUESTED** [review]
We need test code for these utilities. You can start a new test file and use a very small model. Like a 5 bus ED.

File: src/jump_model_utils.jl (hunk 1 above)
Comment (sourabhdalvi) [reply src/jump_model_utils.jl:10]: Yup, I'll add that

Comment (odow) [reply src/jump_model_utils.jl:10]: Don't bother. I approve the release of this under the current PowerSimulations.jl license without the copyright notice.

File: src/jump_model_utils.jl [hunk 2]
```diff
@@ -0,0 +1,157 @@
+# The Numerical stability checks code in this file is based on the code from the SDDP.jl package, 
+ # from the below mentioned commit and file.
+ # commit :8cd305188caffc50a1734913053fc81bba613778 
+ # link to file :https://github.com/odow/SDDP.jl/blob/d353fe5a2903421e7fed6d609eb9377c35d715a1/src/print.jl#L190
+
```
[the remark this answers is not in the document]
Comment (odow) [reply src/jump_model_utils.jl:5]: Woah. How do I do this

[the remark this answers is not in the document]
Comment (jd-lara) [reply src/jump_model_utils.jl:5]: @odow you can copy this: https://github.com/NREL-SIIP/PowerSimulations.jl/blob/master/.github/workflows/format-check.yml

**Review by jd-lara — APPROVED** [review]

**Pull request merged**